query/src/lib/operators.ts

```diff
@@ -148,6 +148,7 @@ export function intersectResults$<
   return map((values) => {
     const isArray = Array.isArray(values);
```

Comment:
these three lines can be outside of the map

Reply:
They all refer to the values parameter, so they cannot?

Reply:
The refetch function needs the toArray variable, so I also cannot cache the refetch function?

Reply:
You can cache it also

Reply:
I cannot cache any of them. All three lines directly or indirectly depend on the values variable, so everything is only available within the map operator.